wp-content/themes/sceraTheatre/templates/homeTemplate.php

```diff
@@ -101,14 +101,52 @@
 		</div>
 	</section>
 
+	<section class="venues-container">
+		<!-- <img class="top-left-border" src="<?php echo get_template_directory_uri(); ?>/img/decoratives/venue-top-corner.png" alt="">
+		<img class="top-right-border" src="<?php echo get_template_directory_uri(); ?>/img/decoratives/venue-top-corner.png" alt=""> -->
+		
+		<div class="title gold">Experience </br> Our Venues</div>
```

Comment:
We should probably make this an `<h2>` instead of a `<div>` And we want to always force the title line break. It doesn't break on desktop (so far).